02/app.js

```diff
@@ -13,11 +14,34 @@ function init() {
 }
 
 function setRandomPosition(element, error = null) {
-    element.style.top = Math.random() * 600 + 'px';
-    element.style.left = Math.random() * 800 + 'px';
+    const alertElement = document.querySelector('.alert');
+    const alertMessage = alertElement.querySelector('.alert__message');
+
+    try {
```

Comment:
Raczej chcemy przechwycić błąd, w miejscu poza jego "deklaracją" - inaczej po prostu zrobilibyśmy to co trzeba w` if`-ie zamiast pisać `throw`. Dlatego w tym przypadku blok `try...catch` w okolicy linii 50